test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -54,15 +63,50 @@ contract FranchiserFactoryHandler is Test {
         factory = _factory;
         votingToken = VotingTokenConcrete(address(factory.votingToken()));
         franchiser = new Franchiser(IVotingToken(address(votingToken)));
-        balances_updated_correctly = true;
-        voting_powers_updated_correctly = true;
+        franchiserLens = new FranchiserLens(IVotingToken(address(votingToken)), factory);
     }
 
     modifier countCall(bytes32 key) {
         calls[key].calls++;
         _;
     }
 
+    // function to increase the account balance mapping of an account by a given amount
+    function _increaseAccountBalance(address _account, uint256 _amount) private {
+        if (!holderAddresses.contains(_account)) {
+            holderAddresses.add(_account);
+            ghost_holders[_account].balance = 0;
+            ghost_holders[_account].votingPower = 0;
+        }
+        ghost_holders[_account].balance += _amount;
+    }
+
+    // function to decrease the account balance mapping of an account by a given amount
+    function _decreaseAccountBalance(address _account, uint256 _amount) private {
+        if (!holderAddresses.contains(_account)) {
+            console2.log("Account not found in holderAddresses on _decreaseAccountBalance");
+        }
+        ghost_holders[_account].balance -= _amount;
+    }
+
+    // function to increase the voting power mapping of an account by a given amount
+    function _increaseAccountVotingPower(address _account, uint256 _amount) private {
+        if (!holderAddresses.contains(_account)) {
```

Comment:
feels a little weird mixing the voting power and the account balance concerns into the same AddressSet and the same struct